common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -96,29 +101,50 @@ private static boolean hasName(Function<?> function, String name) {
         return false;
     }
 
+    private static Optional<Object> getFunctionValueSafely(Function<?> function, String argument) {
+        if (CRASHED_FUNCTIONS.contains(function)) {
+            return Optional.empty();
+        }
+
+        try {
+            Object value = function.getValue(argument);
+            return Optional.ofNullable(value);
+        } catch (Exception e) {
```

Comment:
You probably want to catch Throwable, to get all errors and not only checked exceptions.